src/client/src/components/pages/participants/participantCourseDetails/index.jsx

```diff
@@ -2,31 +2,61 @@ import React, { Component } from "react";
 import { state as initialState, fields as fieldSet } from "./staticData";
 import Form from "./../../../abstract/Form";
 import Footer from "../../../abstract/footer";
+import axios from 'axios';
+import contextHoc from './../../../abstract/HOC/contextHoc';
 
-export default class index extends Component {
+class index extends Component {
```

Comment:
You could use a more descriptive name for your class (like `class ParticipantCourseDetails` instead of `class index`) and then install the [React devtools extension for Chrome](https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi?hl=en), and then you'd see that class name displayed in the component heirarchy when using that new React tab in chrome (you can check out an example screenshot [here](https://lh3.googleusercontent.com/GjX6Q3_FVJfc0DqE2wiPKkgOfth6otzV-D7GV-wB6sH5_t1oodMaHOBLsYOLeydb85bKWu6X=w640-h400-e365)) which can be pretty useful!

Reply:
I was installed it by the way, and yeah you are totally right